pipelines.go

```diff
@@ -80,6 +80,43 @@ func (p Pipeline) String() string {
 	return Stringify(p)
 }
 
+//PipelineTestReport contains detailed information about the results of test run in the pipeline
+type PipelineTestReport struct {
+	TotalTime    int                            `json:"total_time"`
+	TotalCount   int                            `json:"total_count"`
+	SuccessCount int                            `json:"success_count"`
+	FailedCount  int                            `json:"failed_count"`
+	SkippedCount int                            `json:"skipped_count"`
+	ErrorCount   int                            `json:"error_count"`
+	TestSuites   []PipelintTestReportTestSuites `json:"test_suites"`
+}
+
+//PipelintTestReportTestSuites contains test suites test run in the pipeline
+type PipelintTestReportTestSuites struct {
```

Comment:
Hmm... When looking at it again, maybe we can make the names just a little bit shorter:
```suggestion
type PipelintTestSuites struct {
```